app/src/main/java/org/connectbot/HostListActivity.java

```diff
@@ -233,6 +235,7 @@ public boolean onPrepareOptionsMenu(Menu menu) {
 
 		sortcolor.setVisible(!sortedByColor);
 		sortlast.setVisible(sortedByColor);
+		disconnectall.setEnabled(bound.getBridges().size() > 0);
```

Comment:
I think disabling is a better idea for better discovery. Users will know there's a way to disconnect all when not connected to any session.